### PR TITLE
MFTF: Refactoring admin url rewrite multiple storeviews product import with config turned off test

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Description (*)
AdminUrlRewriteMultipleStoreviewsProductImportWithConfigTurnedOffTest is refactored according to the best practices followed by MFTF.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
Run AdminUrlRewriteMultipleStoreviewsProductImportWithConfigTurnedOffTest in the local environment and verified the test runs successfully.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by dmytro-ch** [comment]
@magento run all tests

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminChangeCategoryNameOnStoreViewLevelActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminChangeCategoryNameOnStoreViewLevelActionGroup">
+        <annotations>
+            <description>Updates the Category Name for proper Store View.</description>
+        </annotations>
+        <arguments>
+            <argument name="categoryName" type="string"/>
+        </arguments>
+
+        <uncheckOption selector="{{AdminCategoryBasicFieldSection.categoryNameUseDefault}}" stepKey="uncheckUseDefaultValueENStoreView"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminChangeCategoryNameOnStoreViewLevelActionGroup.xml]: `stepKey` contains misleading information about the store view which is not actually the case in scope of the ActionGroup : )

**Comment by kate-kyzyma** [comment]
Hi @lbajsarowicz 
Thanks for reviewing my test. I have fixed the issue.

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Review by lbajsarowicz — DISMISSED** [review]
:heavy_check_mark: Great job!

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7666](https://jira.corp.magento.com/browse/ENGCOM-7666) has been created to process this Pull Request

**Review by engcom-Alfa — CHANGES_REQUESTED** [review]
Hi @kate-kyzyma . Could you resolve merge conflicts? 
Thanks!

**Comment by kate-kyzyma** [comment]
Hi @engcom-Alfa 
I have resolved the merge conflict.

Thanks.

**Review by lbajsarowicz — APPROVED** [review]
:heavy_check_mark: Great job, thank you for your contribution.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7666](https://jira.corp.magento.com/browse/ENGCOM-7666) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**